src/web-auth/index.js

```diff
@@ -289,6 +289,14 @@ WebAuth.prototype.validateAuthenticationResponse = function(options, parsedHash,
     if (decodedToken.header.alg !== 'HS256') {
```

Comment:
Is there any chance that this get called with a `HS384/512` alg? That also uses "raw secrets" and the warning message below would apply to them as well.